src/site/stages/build/process-cms-exports/tests/assemble-entity-tree.unit.spec.js

```diff
@@ -71,7 +71,23 @@ describe('CMS export', () => {
               path.join(transformedEntitiesDir, transformedFileName),
             ),
           );
-          expect(assembleEntityTree(entity)).to.deep.equal(transformedEntity);
+          let assembled;
+          try {
+            assembled = assembleEntityTree(entity);
+            expect(assembled).to.deep.equal(transformedEntity);
+          } catch (e) {
+            // eslint-disable-next-line no-console
+            console.log(
+              'Transformed entity in the test JSON file:\n',
+              JSON.stringify(transformedEntity, null, 2),
+            );
+            // eslint-disable-next-line no-console
+            console.log(
+              'Transformed from assembleEntityTree:\n',
+              JSON.stringify(assembled, null, 2),
+            );
+            throw e;
```

Comment:
Turns out without this extra logging, it won't log what's different when they're very large objects, so we can't tell what the differences are.